### DISCUSSION TITLE
Upcoming Release of php 8.4 deprecation fix

### DISCUSSION THREAD

**Discussion by ynaut** [Codeception/Codeception#6814] [Q&A] [answered]
Just wanted to ask if there is planned upcoming release which includes the php 8.4 deprecation fix https://github.com/Codeception/Codeception/pull/6811. We are looking to upgrade our application to php 8.4

**Comment by W0rma** [comment, marked as answer]
I have created https://github.com/Codeception/Codeception/issues/6816 which asks for a new release.
